core/camel-main/src/generated/resources/META-INF/camel-main-configuration-metadata.json

```diff
@@ -19,123 +19,123 @@
+    { "name": "camel.main.inflightRepositoryBrowseEnabled", "description": "Defines whether the in-flight repository should allow navigation in each in-flight exchange. This is disabled by default because there is a very slight performance overhead when enabled.", "sourceType": "org.apache.camel.main.DefaultConfigurationProperties", "type": "boolean", "javaType": "boolean", "defaultValue": "false" },
+    { "name": "camel.main.javaRoutesExcludePattern", "description": "Used for exclusive filtering of RouteBuilder classes which are collected from the registry or via classpath scanning. The exclusive filtering takes precedence over inclusive filtering. The pattern is using Ant-path style pattern and multiple patterns can be specified separated by comma. For instance, to exclude all classes starting with Foo , use {code &#42;&#42;\/Foo} . To exclude all routes form a specific package use {code com\/mycompany\/foo\/&#42;} . To exclude all routes form a specific package and its sub-packages use {code com\/mycompany\/foo\/&#42;&#42;} with double wildcards. And to exclude all routes from two specific packages use {code com\/mycompany\/foo\/&#42;,com\/mycompany\/stuff\/&#42;} .", "sourceType": "org.apache.camel.main.DefaultConfigurationProperties", "type": "string", "javaType": "java.lang.String" },
+    { "name": "camel.main.javaRoutesIncludePattern", "description": "Used for inclusive filtering of RouteBuilder classes which are collected from the registry or via classpath scanning. The exclusive filtering takes precedence over inclusive filtering. The pattern is using Ant-path style pattern and multiple patterns can be specified separated by comma. For instance, to include all classes starting with Foo , use {code &#42;&#42;\/Foo} . To include all routes form a specific package use: {code com\/mycompany\/foo\/&#42;} . To include all routes form a specific package and its sub-packages use double wildcards: {code com\/mycompany\/foo\/&#42;&#42;} . And to include all routes from two specific packages use: {code com\/mycompany\/foo\/&#42;,com\/mycompany\/stuff\/&#42;} .", "sourceType": "org.apache.camel.main.DefaultConfigurationProperties", "type": "string", "javaType": "java.lang.String" },
+    { "name": "camel.main.jmxEnabled", "description": "Enables JMX in your Camel application.", "sourceType": "org.apache.camel.main.DefaultConfigurationProperties", "type": "boolean", "javaType": "boolean", "defaultValue": true },
+    { "name": "camel.main.jmxManagementMBeansLevel", "description": "Sets the mbeans registration level. The default value is ManagementMBeansLevel#Default .", "sourceType": "org.apache.camel.main.DefaultConfigurationProperties", "type": "object", "javaType": "org.apache.camel.ManagementMBeansLevel", "defaultValue": "Default" },
```

Comment:
`ManagementMBeansLevel#Default `

I think the full name should be provided or none